src/plugins/discover/public/application/view_components/canvas/index.tsx

```diff
@@ -25,6 +25,32 @@ export default function DiscoverCanvas({ setHeaderActionMenu, history }: ViewPro
     bucketInterval: {},
   });
 
+  const [isCallOutVisible, setIsCallOutVisible] = useState(true);
+  const closeCallOut = () => setIsCallOutVisible(false);
+
+  let callOut;
+
+  if (isCallOutVisible) {
+    callOut = (
+      <EuiFlexItem grow={false}>
+        <EuiCallOut
+          title="You're viewing Discover 2.0. The old Discover app will be retired in OpenSearch version 2.11. To switch back to the old version, toggle the New Discover."
+          iconType="alert"
+          dismissible
+          onDismissible={closeCallOut}
```

Comment:
oh, not sure how i missed this in theOUI PR but this should be `onDismiss` Think we can get that changed in a followup PR?

Reply:
https://github.com/opensearch-project/oui/pull/1003